static/app/views/organizationGroupDetails/groupReplays/groupReplays.tsx

```diff
@@ -0,0 +1,143 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import Link from 'sentry/components/links/link';
+import Pagination from 'sentry/components/pagination';
+import {PanelTable} from 'sentry/components/panels';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {PageContent} from 'sentry/styles/organization';
+import {NewQuery} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {getQueryParamAsString} from 'sentry/utils/replays/getQueryParamAsString';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {useParams} from 'sentry/utils/useParams';
+import ReplayTable from 'sentry/views/replays/replayTable';
+import {Replay} from 'sentry/views/replays/types';
+
+const DEFAULT_DISCOVER_LIMIT = 50;
+
+const GroupReplays = () => {
+  const location = useLocation();
+  const organization = useOrganization();
+  const {selection} = usePageFilters();
+  const params = useParams();
+
+  const getEventView = () => {
+    const {groupId} = params;
+    const eventQueryParams: NewQuery = {
+      id: '',
+      name: '',
+      version: 2,
+      fields: [
+        'eventID',
+        'project',
+        'timestamp',
+        'url',
+        'user.display',
+        'user.email',
+        'user.id',
+        'user.ip_address',
+        'user.name',
+        'user.username',
+      ],
+      projects: selection.projects,
```

Comment:
Can you verify that on the issues stream page, if you have multiple projects selected, click an issue, go to the Replays tab, that this query will only have a single project selected? (the project that the issue belongs to)